hera_sim/tests/test_defaults.py

```diff
@@ -0,0 +1,42 @@
+from os.path import join
+import numpy as np
+from hera_sim.defaults import defaults
+from hera_sim.config import CONFIG_PATH
+from hera_sim.sigchain import gen_bandpass
+from hera_sim.noise import bm_poly_to_omega_p
+
+if defaults._version_is_compatible:
+    def test_config_swap():
+        defaults.set_defaults('h1c')
+        config1 = defaults._config
+        defaults.set_defaults('h2c')
+        assert config1 != defaults._config
+
+    def test_direct_config_path():
+        config = join(CONFIG_PATH, 'HERA_H2C_CONFIG.yaml')
+        defaults.set_defaults(config)
```

Comment:
since you know some of the values in this config, perhaps assert that a few of them are what you expect.